target/linux/ramips/base-files/etc/diag.sh

```diff
@@ -123,8 +123,9 @@ get_status_led() {
 	pbr-d1)
 		status_led="$board:orange:power"
 		;;
+	f5d8235-v1|\
 	f5d8235-v2)
-		status_led="$board:blue:router"
```

Comment:
Why is is the ":blue:router" behaviour unpredictable for failsafe indication? I don't get what you are trying to tell us.

The f5d8235-v2 has no blue wired LED. If this change is really required, please split the f5d8235-v1 and f5d8235-v2 led config.